docs/doc/14-sql-commands/20-query-syntax/06-query-group-by.md

```diff
@@ -1,52 +1,42 @@
 ---
 title: GROUP BY
 ---
+import FunctionDescription from '@site/src/components/FunctionDescription';
 
-The GROUP BY clause in Databend SQL allows you to group rows sharing the same group-by-item expressions and apply aggregate functions to the resulting groups. A group-by-item expression can be a column name, a number referencing a position in the [SELECT](./01-query-select.md) list, or a general expression.
+<FunctionDescription description="Introduced: v1.2.32"/>
```

Comment:
This scope is the GROUP BY, it's weird we support from v1.2.32.

Reply:
Updated the annotation to "Introduced or updated: v1.2.32".